InitializeRepo/initialize.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -euo pipefail
+
+git pull
```

Comment:
Which repo is this pulling?  The `AllSpiceIO/APIDemos` repo?  It seems strange to me that this is part of the script.

Reply:
It pulls whichever repo of the folder you pass it.
If you run this in your AllSpiceIO/APIDemos folder, it will run on that repo.
./initialize.sh .

Reply:
It's the repo of the current working directory, not what you passed it.  `$1` or `$directory` isn't used here.